Cargo.lock

```diff
@@ -1,6 +1,6 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
-version = 4
+version = 3
```

Comment:
Note that I reverted Cargo's `version` bump in the lockfile so that Rust 1.74 continues working.